book/db/README.md

```diff
@@ -0,0 +1,276 @@
+# db
+
+The database is a central component to Reth, enabling persistent storage for data like block headers, block bodies, transactions and more. The Reth database is comprised of key-value storage written to the disk and organized in tables. This chapter might feel a little dense at first, but shortly, you will feel very comfortable understanding and navigating the `db` crate. This chapter will go through the structure of the database, its tables and the mechanics of the `Database` trait.
+
+<br>
+
+## Tables
+
+Within Reth, a table is a struct that implements the `Table` trait.
+
+```rust ignore
+pub trait Table: Send + Sync + Debug + 'static {
+    /// Return table name as it is present inside the MDBX.
+    const NAME: &'static str;
+    /// Key element of `Table`.
+    ///
+    /// Sorting should be taken into account when encoding this.
+    type Key: Key;
+    /// Value element of `Table`.
+    type Value: Value;
+}
+
+//--snip--
+pub trait Key: Encode + Decode {}
+
+//--snip--
+pub trait Value: Compress + Decompress {}
+
+```
+
+The `Table` trait has two generic values, `Key` and `Value`, which need to implement the `Key` and `Value` traits, respectively. There are many tables within the node, all used to store different types of data from `Headers` to `Transactions` and more. Below is a list of all of the tables. You can follow [this link]() if you would like to see the table definitions for any of the tables below.
+
+- CanonicalHeaders
+- HeaderTD
+- HeaderNumbers
+- Headers
+- CumulativeTxCount
+- NonCanonicalTransactions
+- Transactions
+- Receipts
+- Logs
+- PlainAccountState
+- PlainStorageState
+- Bytecodes
+- AccountHistory
+- AccountHistory
```

Comment:
dupe

Reply:
Thanks for catching this, updating it now.